common/components/Header/components/GasPriceDropdown.jsx

```diff
@@ -74,7 +86,11 @@ export default class GasPriceDropdown extends Component {
     });
   };
 
-  updateGasPrice = (e: SyntheticInputEvent) => {
-    this.props.onChange(parseInt(e.target.value, 10));
+  updateGasPrice = (value: string) => {
+    this.props.onChange(parseInt(value, 10));
+  };
+
+  handleGasPriceChange = (e: SyntheticInputEvent) => {
+    this.updateGasPrice(e.target.value);
```

Comment:
you can't directly `throttle` this function, because accessing `SyntheticEvent`s asynchronously is a no-no. See this [page in the React docs](https://facebook.github.io/react/docs/events.html#event-pooling) for more details